doc/md/modules-and-imports.md

```diff
@@ -63,6 +63,37 @@ You can define dependencies for a project using the Vessel package manager or in
 
 In this example, the `Render` module is in the default location for source code in the `redraw` package and the `Mono5x5` module is in a `redraw` package subdirectory called `glyph`.
 
+## Importing packages from a package manager
+
+To download and install third-party packages, a package manager such as [Mops](https://mops.one) or [Vessel](https://github.com/dfinity/vessel) can be used. 
+
+To use either package manager, edit your project's `dfx.json` file to specify a `packtool`, such as:
+
+```json
+{
+  "defaults": {
+    "build": {
+      "packtool": "mops sources"
+    }
+  }
+}
+```
+
+For Vessel, use `vessel sources`. 
+
+Then, download a package with the `mops` or `vessel` CLI tools, such as:
```

Comment:
It might be worth mentioning that for Vessel, it's necessary to edit the `vessel.dhall` file (since there isn't an equivalent to `mops add`). Perhaps we could even just use the same instructions as in [this PR](https://github.com/dfinity/portal/pull/1975/files#diff-2ffe11fc134e5fb5877533a43e097a13b71fd0fdcd974a0e4ab456529b2387e8R11-R51) if that saves any time.